src/components/publishing/icon/edit_embed.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react"
+
+class EditEmbed extends Component<any, null> {
+  render() {
+    return (
+      <svg
+        id="edit-embed"
```

Comment:
Do we want to use className instead here? I _think_ having multiple SVGs with the same id on the page is technically not allowed.

Reply:
good call